js/main.js

```diff
@@ -74,13 +73,14 @@ $searchButton.addEventListener('click', function (event) {
 });
 
 $back.addEventListener('click', function (event) {
+  localStorage.removeItem('search-results');
   searchData = {
     userInput: {},
     astroData: {},
-    photoData: [],
     placesSearchResults: []
   }
   ;
+  sessionStorage.setItem('search-results', searchData);
```

Comment:
Are you intending to use `sessionStorage` here? I don't see you using it anywhere else 🤔

Reply:
No, I forgot to delete it!